docs/ref/checks.txt

```diff
@@ -319,6 +319,14 @@ with the admin site:
 * **admin.E035**: The value of ``readonly_fields[n]`` is not a callable, an
   attribute of ``<ModelAdmin class>``, or an attribute of ``<model>``.
 
+The following checks are performed on the default 
+:class:`~django.contrib.admin.AdminSite`
+
+* **admin.E401**: :mod:``django.contrib.contenttypes`` must be in 
+  INSTALLED_APPS in order to use the admin application
```

Comment:
Please add periods here and in the actual messages. Also add :setting: links for the settings here.